### ISSUE TITLE
Failed to load the ide-crystal package

### ISSUE THREAD

**Issue by AlexWayfer** [crystal-lang-tools/atom-ide-crystal#31]
[Enter steps to reproduce:]

1. Start Atom Editor with installed package.

**Atom**: 1.25.0 x64
**Electron**: 1.7.11
**OS**: linux 4.15.12-1-ARCH
**Thrown From**: [ide-crystal](https://github.com/crystal-lang-tools/atom-ide-crystal) package 0.3.3

### Stack Trace

Failed to load the ide-crystal package

```
At Unexpected token import

SyntaxError: Unexpected token import
    at Module.get_Module._compile (<embedded>:11565:52)
    at Object.value [as .ts] (<embedded>:11829:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (<embedded>:96:26)
    at Package.requireMainModule (<embedded>:73045:33)
    at measure (<embedded>:72406:22)
    at Package.measure (<embedded>:72351:25)
    at Package.load (<embedded>:72392:16)
    at PackageManager.loadAvailablePackage (<embedded>:25658:16)
    at config.transact (<embedded>:25571:20)
    at Config.transact (<embedded>:22331:20)
    at PackageManager.loadPackages (<embedded>:25569:23)
    at loadStatePromise.loadState.then (<embedded>:1069:27)
    at <anonymous>:null:null

```

### Commands

```
```

### Non-Core Packages

```
advanced-open-file 0.16.8 
atom-beautify 0.32.2 
atom-material-syntax 1.0.8 
atom-material-ui 2.1.3 
atom-smart-tabs 0.1.0 
atom-tabs-in-sidebar 1.1.0 
atom-typescript 12.3.8 
auto-detect-indentation 1.3.0 
auto-update-plus 0.5.6 
autoclose-html 0.23.0 
autocomplete-html-entities 0.2.0 
autocomplete-paths 2.12.2 
autocomplete-ruby 0.2.8 
autoupdate-packages 1.3.1 
busy-signal 1.4.3 
color-picker 2.2.5 
editorconfig 2.2.2 
genesis-ui 0.5.0 
git-time-machine 1.5.9 
goto-definition 1.3.4 
highlight-selected 0.13.1 
ide-crystal 0.3.3 
intentions 1.1.5 
language-crystal-actual 0.37.2 
language-fish-shell 1.1.0 
language-haml 0.25.2 
language-vue 0.23.1 
linter 2.2.0 
linter-erb 1.1.0 
linter-eslint 8.4.1 
linter-htmlhint 1.4.0 
linter-jshint 3.1.6 
linter-markdown 5.2.0 
linter-reek 2.2.2 
linter-rubocop 2.2.3 
linter-ruby 1.2.6 
linter-spell 0.15.0 
linter-stylelint 4.2.0 
linter-ui-default 1.7.1 
merge-conflicts 1.4.5 
minimap 4.29.8 
minimap-highlight-selected 4.6.1 
monocolor-blue-syntax 2.0.0 
monokai 0.24.0 
nord-atom-syntax 0.9.1 
northem-atom-syntax 2.1.0 
northem-dark-atom-syntax 2.1.0 
nucleus-dark-ui 0.12.3 
platformio-ide-terminal 2.8.1 
remember-file-positions 0.2.3 
revert-buffer 0.6.0 
ruby-block 0.3.6 
ruby-define-method 0.2.2 
ruby-test 1.0.2 
seeing-is-believing 15.0.0 
seti-icons 1.5.4 
seti-ui 1.10.0 
single-click-open 0.2.2 
space-tab 0.4.2 
space-underscore 0.1.1 
split-diff 1.5.2 
stacked-tabs 0.1.3 
symbols-tree-view 0.14.0 
sync-settings 0.8.5 
synced-sidebar 0.4.3 
tab-control 0.6.10 
tabs-to-spaces 1.0.3 
tasks 2.10.1 
term3 0.22.1 
toggle-quotes 1.1.0 
tree-view-open-files 0.3.0 
vertabs 1.1.1 
yard 0.5.1 
zentabs 0.8.9 
```

**Comment by keplersj** [comment]
Looks like it's not transpiling the TypeScript source of this package correctly. Does this still happen after uninstalling and reinstalling this package?

**Comment by AlexWayfer** [comment]
> Does this still happen after uninstalling and reinstalling this package?

Yes. :frowning_face:

**Comment by keplersj** [comment]
Huh. Help, @smhxx?

**Comment by smhxx** [comment]
I believe this is a regression in v1.5.0 of `atom-ts-transpiler`, I've unpublished for the time being until I can determine what the bug is and how it managed to slip through my integration testing. If you uninstall `ide-crystal`, restart Atom, and reinstall the package, it should resume working normally. This should not affect users who installed `ide-crystal` prior to the last hour or so.

**Comment by AlexWayfer** [comment]
> If you uninstall ide-crystal, restart Atom, and reinstall the package, it should resume working normally.

Thanks, it helped. Restarting Atom was required for success before reinstalling. I didn't do this in a previous attempt:

> > Does this still happen after uninstalling and reinstalling this package?
>
> Yes. :frowning_face:

**Comment by smhxx** [comment]
It wouldn't have helped anyway, until about 10 minutes ago when I unpublished. The problem was that the broken version of `atom-ts-transpiler` (v1.5.0) was in-range and thus was being "preferred" over v1.4.3 for new installs. v1.4.3 is now marked as `latest` again, so re-installing reverts to the previous, working version of the package.

**Comment by smhxx** [comment]
I've just manually published `atom-ts-transpiler@1.5.1` which fixes the issues that had been present in v1.5.0. This should be safe to close now as I'm confident that the problem was external to `ide-crystal` and I've confirmed on my end that I no longer experience this bug with the latest version of the transpiler.

**Comment by AlexWayfer** [comment]
> I've just manually published atom-ts-transpiler@1.5.1 which fixes the issues that had been present in v1.5.0. This should be safe to close now as I'm confident that the problem was external to ide-crystal and I've confirmed on my end that I no longer experience this bug with the latest version of the transpiler.

Okay, thank you!

Issue can be re-opend in case of regression, I think.
